lib/did_you_mean/word_collection.rb

```diff
@@ -1,3 +1,4 @@
+require 'ostruct'
```

Comment:
I originally build OpenStructs instead of Arrays, but eventually decided new arrays would be faster than new objects, so I took the OpenStucts out, but forgot to remove the require.